EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 7664
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.